charts/localstack/values.yaml

```diff
@@ -149,3 +152,13 @@ nodeSelector: {}
 tolerations: []
 
 affinity: {}
+
+# Mount /docker-entrypoint-initaws.d to run startup scripts with
+# {{ template "localstack.fullname" . }}-init-scripts-config configMap
+enableStartupScripts: true
+
+# Add startup scripts content used by {{ template "localstack.fullname" . }}-init-scripts-config configMap
+# to run at localstack startup
+startupScriptContent: |
+  # Add startup scripts content used by {{ template "localstack.fullname" . }}-init-scripts-config configMap 
```

Comment:
I'd also leave this empty by default. You can use the docstring above to describe how it can work..